packages/mathjax/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@jupyterlab/mathjax2",
+  "name": "@jupyterlab/mathjax",
   "version": "4.0.0-alpha.17",
   "description": "A MathJax 2 Typesetting provider for JupyterLab",
```

Comment:
Maybe the `description` field could be updated to either mention MathJax 3 or no version at all?